internal/xds/v3/callbacks.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v3
+
+import (
+	"fmt"
+
+	envoy_service_discovery_v3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	envoy_server_v3 "github.com/envoyproxy/go-control-plane/pkg/server/v3"
+	"github.com/sirupsen/logrus"
+)
+
+func NewCallbacks(log *logrus.Logger) envoy_server_v3.Callbacks {
```

Comment:
nit: Need a comment describing this function.

Reply:
Quick drive-by follow up ... we should name this something more specific too :)

Reply:
`NewEnvoyXDSCallbacks` (maybe stick `Server` somewhere in there)?

Reply:
This specifically gives you logging, so maybe `RequestLoggger`, `RequestLoggingCallbacks`?